index.d.ts

```diff
@@ -0,0 +1,12 @@
+type Loggable = string | number | {};
```

Comment:
It's hard to say. We currently can log anything we want. However I can think of use cases for the above, and possibly arrays. Could just switch to any?